helmfile.d/00-ccd-shared-database.yaml

```diff
@@ -5,6 +5,7 @@ repositories:
 releases:
   - name: ccd-shared-database
     chart: bitnami/postgresql
+    version: 10.16.2
```

Comment:
Seems ok to me, but did something break with taking the latest version?

Reply:
The lates version caused some problems in our local. So Alex, Gareth and me talked in channel about this. This change applied by @sullivangareth before via this PR 
https://github.com/hmcts/wa-kube-environment/pull/98/files